tests/test_tests_support_assertover.py

```diff
@@ -0,0 +1,41 @@
+import unittest
```

Comment:
Missing license header. I'll add one here, but perhaps you could make a PR to do the same in the test_support, test_boolean and any other `tests` modules where I've missed them?